### DISCUSSION TITLE
[Feature] Automatically detect or ask for what language a user is using before the title screen to prevent region crashes

### DISCUSSION THREAD

**Discussion by kalomaze** [Raytwo/ARCropolis#308] [Ideas] [answered]
First time non-english users of mods like HDR that use region specific file replacement (like the title screen) will crash unless they turn off ARCropolis to switch to English in vanilla, then change their ARCropolis config to their actual language, and then finally changing their game language back in vanilla. This is a one time fix for whoever has the issue, but it still leads to a lot of confusion for the uninformed

[HDR title screen.zip](https://github.com/Raytwo/ARCropolis/files/8501068/HDR.title.screen.zip)

- How should it be implemented?

During first time boot, run a check to see the language setting based on the region of the files the game calls in or potentially via save file if that's not impossible. Alternatively, redirect every first time installer of ARCropolis to the language settings so they can fill this in manually, if automatic correction isn't feasible

**Comment by Raytwo** [comment, marked as answer]
As told in DMs, this already exists as of ARCropolis 3.2.0
